src/components/backoffice/auction/formConfig.tsx

```
@@ -0,0 +1,101 @@
+
+export const productConditionOptions = [
+    { value: "BEST", label: "최상" },
+    { value: "GREAT", label: "상" },
+    { value: "GOOD", label: "중" },
+];
+
+// --- 폼 구조를 데이터로 정의 ---
+// 폼의 각 행과 필드를 객체 배열로 설정하여, JSX를 직접 수정하지 않고 이 배열만으로 폼을 동적으로 생성합니다.
+
+export interface FormField {
+  component: React.ComponentType<any>;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)